ang/civicase/CaseOverview.html

```diff
@@ -31,6 +54,13 @@
           </div>
           <span class="civicase__case-overview__flow-status__border"  ng-style="{'background-color': status.color}"></span>
         </div>
+        <div class="civicase__case-overview__flow-status" ng-if="isAllStatusesDisbabled()">
+          <div class="civicase__case-overview__flow-status__no-status-selected-warning">
```

Comment:
Class name is too big. How about `civicase__case-overview__flow-status__empty-state`?